src/pegout/components/RbtcInputAmount.vue

```diff
@@ -147,6 +147,8 @@ export default defineComponent({
         || safeAmount.value.lt(minValue);
     });
 
+    const clearProp = computed(() => props.clear);
```

Comment:
Could we skip the computed property? Watching `() => props.clear` directly should accomplish the same thing without the extra step.

Reply:
done! thanks